src/main/java/ch/njol/skript/util/FileUtils.java

```diff
@@ -60,7 +63,24 @@ public static String getBackupSuffix() {
 			return "" + backupFormat.format(System.currentTimeMillis());
 		}
 	}
-	
+
+	public static void backupPurge(File f, int toKeep) throws IOException, IllegalArgumentException {
```

Comment:
what's f for? f doesn't tell me anything. give it a better name
also this method should have javadocs